docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -5,531 +5,129 @@
 
-        $ ansible myhost --become -m raw -a "yum install -y python2"
+.. _getting_ansible:
 
 .. _what_version:
 
-Selecting an Ansible artifact and version to install
+Selecting an Ansible package and version to install
 ====================================================
 
-Starting with version 2.10, Ansible distributes two artifacts: a community package called ``ansible`` and a minimalist language and runtime called ``ansible-core`` (called `ansible-base` in version 2.10). Choose the Ansible artifact and version that matches your particular needs.
-
-Installing the Ansible community package
-----------------------------------------
-
-The ``ansible`` package includes the Ansible language and runtime plus a range of community curated Collections. It recreates and expands on the functionality that was included in Ansible 2.9.
-
-You can choose any of the following ways to install the Ansible community package:
+Ansible's community packages are distributed in two ways: a minimalist language and runtime package called ``ansible-core``, and a much larger "batteries included" package called ``ansible``, which adds a community-curated selection of :ref:`Ansible Collections <collections>` for automating a wide variety of devices. Choose the package that fits your needs; the following instructions will refer to ``ansible``, but ``ansible-core`` can be substituted if you prefer to start with a more minimal package and install only the Ansible Collections you require.
 
-* Install the latest release with your OS package manager (for Red Hat Enterprise Linux (TM), CentOS, Fedora, Debian, or Ubuntu).
-* Install with ``pip`` (the Python package manager).
 
-.. _install_core:
+Installing and upgrading Ansible
+================================
 
-Installing `ansible-core`
--------------------------
+Locating Python
+---------------
 
-Ansible also distributes a minimalist object called ``ansible-core`` (or ``ansible-base`` in version 2.10). It contains the Ansible language, runtime, and a short list of core modules and other plugins. You can build functionality on top of ``ansible-core`` by installing collections from Galaxy, Automation Hub, or any other source.
+Locate and remember the path to the Python interpreter you wish to use to run Ansible. We'll refer to this Python in the following instructions as ``python3``. For example, if you've determined that you want the Python at ``/usr/bin/python3.9`` to be the one that you'll install Ansible under, specify that instead of ``python3``.
```

Comment:
```suggestion
Locate and remember the path to the Python interpreter you wish to use to run Ansible. The following instructions refer to this Python  as ``python3``. For example, if you've determined that you want the Python at ``/usr/bin/python3.9`` to be the one that you'll install Ansible under, specify that instead of ``python3``.
```